src/app.js

```diff
@@ -18,16 +18,28 @@
 	
 	/** Mediator **/
 	var mediatorIsLoadingPage = false;
-	var currentRouteUrl = document.location.href.substring(
-		document.location.protocol.length + 2 + document.location.host.length
-	);
+	var currentRouteUrl = getCurrentUrl();
```

Comment:
The function is not defined at this point in the code.

Reply:
Ah damn.

Reply:
@fhamon Check 132b91d611